spec/models/user_rank_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper' 
+describe UserRank do
+  it 'returns the correct rank and message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.